### ISSUE TITLE
ScreenCapture on macOS?

### ISSUE THREAD

**Issue by diederikh** [shogo4405/lf.swift#63]
I am right that screen capture (`httpStream.attachScreen(ScreenCaptureSession())` does not work on macOS?

**Comment by shogo4405** [comment]
Not support yet macOS.

**Comment by diederikh** [comment]
I'll see if I can implement this part.

**Comment by agquick** [comment]
Any progress here? I also need this. Let me know if I can help.

**Comment by diederikh** [comment]
I have a version working for macOS. It's in my fork. I still have to do some cleanup and refactoring before I can do a PR.

**Comment by shogo4405** [comment]
Hi. framework will support for ScreenCapture on macOS next release 0.3.4.
macOS has the ScreenCapture api "AVCaptureScreenInput". https://developer.apple.com/library/mac/documentation/AVFoundation/Reference/AVCaptureScreenInput_Class/

snippet code is here.
```swift
let screen = AVCaptureScreenInput(displayID: CGMainDisplayID())
rtmpStream.attachScreen(screen)
```

**Comment by shogo4405** [comment]
released 0.3.4
